packages/gator-permissions-controller/CHANGELOG.md

```diff
@@ -7,11 +7,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.4.0]
+
 ### Added
 
-- Add `submitRevocation` action to submit permission revocations through the gator permissions provider snap
-- Add `addPendingRevocation` action to queue revocations until transaction confirmation
-- Expose list of pending revocations in state
+- New Action on the GatorPermissionsController to Allow the Consumer to SubmitRevocation ([#6713](https://github.com/MetaMask/core/pull/6713))
```

Comment:
1. It looks like there are two actions introduced, not just one. These are also available as methods.
2. What's the new action called? It would be helpful to be more specific so that in reviewing this changelog in the future we don't have to go digging through PRs.
3. A new type was added to the list of exports.

Perhaps this?

``` suggestion
- Add `submitRevocation` and `addPendingRevocation` methods to GatorPermissionsController ([#6713](https://github.com/MetaMask/core/pull/6713))
  - These are also available as actions (`GatorPermissionsController:submitRevocation` and `GatorPermissionsController:addPendingRevocation`)
- Export `RevocationParams` type ([#6713](https://github.com/MetaMask/core/pull/6713))